src/App.vue

```diff
@@ -0,0 +1,32 @@
+<template>
+  <div>
+    <h1>Todo List</h1>
+    <AddTodo />
+    <hr>
+    <todoList />
```

Comment:
Think if you lead the development if this project, how can you enforce name consistency for components? `<AddTodo />` `  <todoList />`

Reply:
My bad. No, its not acceptable names for components. I think, it should be called like "addTaskComponent" and "tasksListComponent"

Reply:
I mean `camelCase` vs `PascalCase`. How do you enforce so everyone use a specific one?